bylaws/001-mission-and-structure.md

```diff
@@ -55,13 +55,13 @@ There are a number of defined functions that the FIG Secretaries are expected to
 * Tallying votes and managing the voting system
 * Tracking member project activity
 * Ensuring bylaws are being followed
-* Clarifying any interpretation of bylaw text (If there is lack of consensus between FIG Secretaries then it must be put to a full vote); subject to a lack of objection from Core Committee members and Project Representatives**
+* Clarifying any interpretation of bylaw text; subject to a lack of objection from Core Committee members and Project Representatives**
```

Comment:
semicolon should be comma.  But since that's a very long list of people who can object, this is basically not an authority in practice, no?  Anyone objects, and then... what?